pkg/webhooks/validator/validator.go

```diff
@@ -481,6 +506,32 @@ func (wh *WebhookHandler) validateDeprecatedFeatureGates(hc *v1beta1.HyperConver
 	return warnings
 }
 
+func (wh *WebhookHandler) validateAffinity(hc *v1beta1.HyperConverged) error {
+	if hc.Spec.Workloads.NodePlacement != nil {
+		if err := validateAffinity(hc.Spec.Workloads.NodePlacement.Affinity); err != nil {
+			return fmt.Errorf("invalid workloads node placement affinity: %v", err.Error())
+		}
+	}
+
+	if hc.Spec.Infra.NodePlacement != nil {
+		if err := validateAffinity(hc.Spec.Infra.NodePlacement.Affinity); err != nil {
+			return fmt.Errorf("invalid infra node placement affinity: %v", err.Error())
+		}
+	}
+
+	return nil
+}
+
+func validateAffinity(affinity *corev1.Affinity) error {
+	if affinity == nil || affinity.NodeAffinity == nil || affinity.NodeAffinity.RequiredDuringSchedulingIgnoredDuringExecution == nil {
```

Comment:
users can use the other affinity rule - `preferredDuringSchedulingIgnoredDuringExecution`, why we won't accept it?